changelog.md

```diff
@@ -10,54 +10,54 @@ class: changelog
 
 ## Version 5.0.2 March 5, 2019
 * Added presentation and document presets to `htmlpanel` dialog component.
-* Added missing fixed_toolbar_container setting has been reimplemented in the Silver theme.
-* Added a new toolbar setting `toolbar_drawer` that moves toolbar groups which overflow the editor width into either a `sliding` or `floating` toolbar section.
-* Updated the build process to include package lock files in the dev distribution archive.
-* Fixed inline dialogs did not have aria attributes.
-* Fixed default icons are now available in the UI registry, allowing use outside of toolbar buttons.
+* Added missing `fixed_toolbar_container` setting that has been reimplemented in the Silver theme.
+* Added a new toolbar setting `toolbar_drawer` that moves toolbar groups which overflow the editor width into either a *sliding* or *floating* toolbar section.
+* Changed the build process to include package lock files in the dev distribution archive.
+* Fixed inline dialogs that did not have aria attributes.
+* Fixed default icons, now available in the UI registry, to allow use outside of toolbar buttons.
```

Comment:
This doesn't quite sound right. I can't quite say what's wrong but it seems to change the original meaning. Maybe `Fixed default icons not being available in the UI registry, to allow use outside of toolbar buttons.` or something? The key point of this change, is that the default icons weren't in the registry and now they are. The last part is just useful information.

Reply:
Reworded sentence for technical accuracy.